test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -48,7 +48,7 @@ const (
 	kubeSystemPodsReadinessChecks             = 6
 	sleepBetweenRetriesWhenWaitingForPodReady = 1 * time.Second
 	timeoutWhenWaitingForPodOutboundAccess    = 1 * time.Minute
-	stabilityCommandTimeout                   = 5 * time.Second
+	stabilityCommandTimeout                   = 1 * time.Second
```

Comment:
These are the commands that are re-using this const:

- `nc -vz 8.8.8.8 53 || nc -vz 8.8.4.4 53`
- `nc -vz kubernetes 443 && nc -vz kubernetes.default.svc 443 && nc -vz kubernetes.default.svc.cluster.local 443`
- `nc -vz apache-pod.default.svc.cluster.local 80`
- `nc -vz bbc.co.uk 80 || nc -vz google.com 443 || nc -vz microsoft.com 80`

I think it's fair to say that a 1 second execution duration of any of the above is "interesting" and we'd want our stability tests to fail as a result.